### PR TITLE
Fix CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by shiannn**
Fix the "Setup" section in `CONTRIBUTING.md` by replacing `USAGE.md` to `docker_install.md`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Contributors should develop on [their own fork](https://help.github.com/en/githu
 ## Setup
 Go to https://github.com/quic/aimet and fork the repo using [these instructions](https://help.github.com/en/github/getting-started-with-github/fork-a-repo).
 
-Follow the [Requirements](USAGE.md#requirements) and [Setup](USAGE.md#setup-the-environment) sections in [USAGE.md](USAGE.md). Then return to this page.
```
Comment (shiannn) [inline CONTRIBUTING.md:38]: Currently `USAGE.md` is renamed to `docker_install.md`

**Comment by shiannn** [comment]
Fix #2022

**Review by quic-mangal — APPROVED** [review]

**Review by quic-akhobare — APPROVED** [review]

**Comment by quic-akhobare** [comment]
Thanks @shiannn. The job failing is unrelated to you. And is being looked at. We should have a resolution for that soon I believe.

**Review by quic-bharathr — APPROVED** [review]
Thanks @shiannn ! We'll merge it shortly after we fix the PR status check jobs.

**Pull request merged**